lib/src/main/java/uk/gov/di/ipv/cri/kbv/api/library/domain/ParseJWT.java

```diff
@@ -0,0 +1,69 @@
+package uk.gov.di.ipv.cri.kbv.api.library.domain;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.nimbusds.jwt.SignedJWT;
+
+import java.text.ParseException;
+import java.time.LocalDate;
+import java.util.List;
+import java.util.Optional;
+import java.util.stream.Collectors;
+
+public class ParseJWT {
+
+    public static final String CLAIMS = "claims";
+    public static final String VC_HTTP_API = "vc_http_api";
+    private final ObjectMapper objectMapper;
+
+    public ParseJWT(ObjectMapper objectMapper) {
+        this.objectMapper = objectMapper;
+    }
+
+    public Optional<PersonIdentity> getPersonIdentity(String jwt)
+            throws ParseException, JsonProcessingException {
+        try {
```

Comment:
I don't think you need the try { } catch {} since you are throwing in method declaration 

```
    public Optional<PersonIdentity> getPersonIdentity(String jwt)
            throws ParseException, JsonProcessingException {
```

Reply:
Removed.